R-package/NEWS.md

```diff
@@ -0,0 +1,28 @@
+## RGF 1.0.4
+
+* We (RGF-team) modified the license from GPL-3 to MIT to go in accordance with the new structure of the *rgf_python* package. The package includes two files : *LICENSE* (for the RGF R package) and *LICENSE.note* (for the *RGF*, *FastRGF* and *rgf_python* packages).
```

Comment:
I think we don't need this clarification anymore: `We (RGF-team) modified` -> `We modified`